client/spec/search_spec.js

```diff
@@ -71,4 +73,40 @@ describe('Search', function() {
         priority3.click();
         expect(element.all(by.repeater('items._items')).count()).toBe(1);
     });
+
+    it('can search by from desk field', function() {
+        workspace.switchToDesk('SPORTS DESK').then(content.setListView);
+        expect(element.all(by.repeater('items._items')).count()).toBe(2);
+        authoring.createTextItem();
+        authoring.writeTextToHeadline('From-Sports-To-Politics');
+        authoring.writeText('This is Body');
+        authoring.writeTextToAbstract('This is Abstract');
+        authoring.save();
+        expect(element.all(by.repeater('items._items')).count()).toBe(3);
+        authoring.sendTo('Politic Desk');
+        authoring.confirmSendTo();
+        workspace.switchToDesk('POLITIC DESK').then(content.setListView);
+        expect(element.all(by.repeater('items._items')).count()).toBe(8);
+        globalSearch.openGlobalSearch();
+        globalSearch.setListView();
+        expect(element.all(by.repeater('items._items')).count()).toBe(11);
+        globalSearch.openFilterPanel();
+        globalSearch.openParameters();
+
+        globalSearch.selectDesk('from-desk', 'Sports Desk');
+        expect(element.all(by.repeater('items._items')).count()).toBe(1);
+        expect(globalSearch.getHeadlineElement(0).getText()).toBe('From-Sports-To-Politics');
+
+        globalSearch.selectDesk('to-desk', 'Politic Desk');
+        expect(element.all(by.repeater('items._items')).count()).toBe(1);
+        expect(globalSearch.getHeadlineElement(0).getText()).toBe('From-Sports-To-Politics');
+
+        globalSearch.selectDesk('from-desk', '');
+        expect(element.all(by.repeater('items._items')).count()).toBe(1);
+        expect(globalSearch.getHeadlineElement(0).getText()).toBe('From-Sports-To-Politics');
+
+        globalSearch.selectDesk('to-desk', '');
+        expect(element.all(by.repeater('items._items')).count()).toBe(11);
```

Comment:
Would it make sense to extract the `element.all(by.repeater('items._items'))` part into a helper object/function? It is repeated a lot throughout the code...